ci_support/run_docker_build.sh

```diff
@@ -37,16 +39,11 @@ echo "$config" > ~/.condarc
 # A lock sometimes occurs with incomplete builds. The lock file is stored in build_artefacts.
 conda clean --lock
 
+conda update --yes --all
+#conda install --yes conda-build==1.18.2
```

Comment:
I see. Slightly worrying that some feedstocks *require* one version of conda-build and other feedstocks *require* another version.

Anyway, :+1: